.env.dist

```diff
@@ -0,0 +1,7 @@
+UID=1000
+GID=1000
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+PHP_IDE_CONFIG=serverName=my-function
```

Comment:
FIX: Newline at the end of the file.

This is a habit I picked up back in uni from an anal retentive professor but certain tooling seems to be equally anal.

Note all the diffs that have no newline have a (-) marker to show the lack of one.  That's pretty distracting and my OCD wants the new line as well :)